src/security/content-security-policy.conf

```diff
@@ -8,18 +8,66 @@
 # This can be done by setting a `Content Security Policy` which
 # whitelists trusted sources of content for your website.
 #
-# The example header below allows ONLY scripts that are loaded from
-# the current website's origin (no inline scripts, no CDN, etc).
-# That almost certainly won't work as-is for your website!
+# There is no one policy that fits all websites, you will have to modify
+# the `Content-Security-Policy` directives in the example below depending
+# on your needs.
 #
-# To make things easier, you can use an online CSP header generator
-# such as: https://www.cspisawesome.com/.
+# The example policy below ensures that:
 #
+#  (1) All resources are restricted to the origin of the current website
```

Comment:
I don't think that's true. There are directives that don't inherit from `default-src`.

Reply:
It is true for the directives you use, but generally it's not true.

Reply:
Indeed. That's also why [I suggested](https://github.com/h5bp/server-configs-apache/pull/181#discussion_r266406669) to reduce verbosity of this documentation and prefer linking to official and rigorous docs already listed. What do you think?

Reply:
@XhmikosR @LeoColomb 

I've updated for less verbosity to address both those statements (also cleaned up indentation) in https://github.com/h5bp/server-configs-apache/pull/181/commits/14b3aa98dd12bbf105981ad35a09600d2cadaf4d.

Thanks for the input, don't hesitate to tell me if I missed something.